### ISSUE TITLE
angular reactive forms

### ISSUE THREAD

**Issue by pasblin** [guillotinaweb/ngx-schema-form#402]
Hi, 
it is possible to use this library using angular reactive  forms without using ngmodel?

regards

**Comment by krishna217** [comment]
yes it is. Just use `formcontrol` instead of ngModel with `sf-form`
